### PR TITLE
Docusaurus improvements

### PR DESCRIPTION
**Pull request by ShogunPanda**

### CODE REVIEW

File: packages/docs/pages/usage/insert.mdx [hunk 1]
```diff
@@ -197,3 +197,81 @@ the document will be indexed with the following `id`: `73cbcc79-2203-49b8-bb52-6
...
 <Callout type="warning">If you try to insert two documents with the same ID, Orama will throw an error.</Callout>
+
+## Remote document storing
+
+By default Orama keeps a copy of the inserted document in memory (and in the serialized data) to speed up search performance.
+
+If this is not acceptable, you can provide a custom `documentsStore` components which will be responsible to store
+and fetch documents from another location (local or remote).
+
+The code example below is a blueprint of a possible implementation.
```
Comment (allevo) [inline packages/docs/pages/usage/insert.mdx:208]: Be more practical:

```suggestion
The code example below is an example that implements a proxy: when a document is requested, the code finds it on a location of the filesystem.
You can replace the file reading with your custom code.
```

Could you change the doce below accordingly? Something like
```
JSON.parse(await fs.readFile(path.join(ROOT_LOCATION, id)))
```

Could you put a note for the performance issue?

File: packages/docs/pages/usage/insert.mdx [hunk 2]
```diff
@@ -197,3 +197,81 @@ the document will be indexed with the following `id`: `73cbcc79-2203-49b8-bb52-6
...
+        return yourDataSource.get(id)
+      },
+      getMultiple() {
+        /*
+          The ID argument is the final ID chosen by Orama for the document, as described in the
+          "Custom document IDs" above.
+
+          The function below can return a promise.
+        */
+        return yourDataSource.getMultiple(id)
```
Comment (allevo) [inline packages/docs/pages/usage/insert.mdx:242]: where id came from?

File: packages/docs/pages/usage/insert.mdx (hunk 1 above)
Comment (micheleriva) [reply packages/docs/pages/usage/insert.mdx:208]: +1

File: packages/docs/pages/usage/insert.mdx (hunk 2 above)
Comment (ShogunPanda) [reply packages/docs/pages/usage/insert.mdx]: Fixed.

File: packages/docs/pages/usage/insert.mdx (hunk 1 above)
Comment (ShogunPanda) [reply packages/docs/pages/usage/insert.mdx]: Fixed.

**Review by allevo — APPROVED** [review]
LGTM

**Review by micheleriva — APPROVED** [review]
lgtm

**Pull request merged**